src/python/pants/testutil/pants_integration_test.py

```diff
@@ -114,6 +114,9 @@ def run_pants_with_workdir_without_waiting(
 
     if hermetic:
         args.append("--pants-config-files=[]")
+        # Certain tests may be invoking `./pants test` for a pytest test with conftest discovery
+        # enabled. We should ignore the root conftest.py for these cases.
+        args.append('--pants-ignore=["/conftest.py"]')
```

Comment:
Clever!

Reply:
I have a good idea every now and then :stuck_out_tongue: